src/Impostor.Patcher/Impostor.Patcher.Shared/AmongUsModifier.cs

```diff
@@ -143,7 +143,10 @@ public async Task<bool> SaveIpAsync(string input)
                 return false;
             }
 
-            return WriteIp(ipAddress, port);
+            var result = WriteIp(ipAddress, port);
```

Comment:
What's the logic behind the changes in this file?

Reply:
https://github.com/Impostor/Impostor/pull/235